Modules/Filtering/LabelMap/include/itkLabelShapeKeepNObjectsImageFilter.hxx

```diff
@@ -29,6 +29,7 @@ LabelShapeKeepNObjectsImageFilter<TInputImage>::LabelShapeKeepNObjectsImageFilte
   m_NumberOfObjects = 1;
   m_ReverseOrdering = false;
   m_Attribute = LabelObjectType::NUMBER_OF_PIXELS;
+  this->ThreaderUpdateProgressOff();
```

Comment:
LabelShaeKeepNObjectsImageFilter is also derived from InPlaceLabelMap/LabelMapFilter